internal/host/common.go

```diff
@@ -74,6 +74,21 @@ var manualRebootStages = []models.HostStage{
 	models.HostStageDone,
 }
 
+var allStages = []models.HostStage{
```

Comment:
Should we call this `installationStages` ?

IIUC, this slice does not contain *all* the stages that a host can be in, does it?

Reply:
It does contains all the values of the `models.HostStage` enum. If I understand correctly all the values of that enum are installation stages, and there are no other stages. Note that there is not the host _state_, but the _stage_ which is only used to report the progress of the installation.

Reply:
Maybe it will be worth looking at the state?
if the agent is the one that reporting the stage then we probably don't want to update when host is preparing for installation is well no?

Reply:
Hmmm, tricky one.
On the one hand, it's true that all stages are actually installation stages (we report the stage only during the installation), so the name is accurate. 
On the other hand, when you see the validation is marked with `skippedStates: allStages` it seems like it's always skipped (and not just during installations).
I guess a comment explaining that the stage is set only during installation should resolve it.

Reply:
> Maybe it will be worth looking at the state?
True

Reply:
as you said it will resolve most of the cases, and we can fix the other cases later